src/components/Quote.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+import './Quote.css';
+
+function Quote() {
+  const [quoteData, setQuoteData] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    fetch('https://api.api-ninjas.com/v1/quotes?category=happiness', {
+      headers: {
+        'X-Api-Key': 'oF/Wh13s4YEHx1jaPyWlmw==eYn2l3wmuDgNIs36',
+      },
+    })
+      .then((response) => {
+        if (response.ok) return response.json();
+        throw response;
+      })
+      .then((data) => {
+        setQuoteData(data[0]); // We only need the first quote
```

Comment:
- [ ] [OPTIONAL] Kindly note that comments are not supposed to duplicate codes, you can remove the statement, ` // We only need the first quote`. Check out [this](https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/) article for more.